vmm/src/lib.rs

```diff
@@ -2144,10 +2147,10 @@ const DEVICE_MANAGER_SNAPSHOT_ID: &str = "device-manager";
 mod unit_tests {
     use super::*;
     #[cfg(target_arch = "x86_64")]
-    use crate::config::DebugConsoleConfig;
-    use config::{
-        ConsoleConfig, ConsoleOutputMode, CpusConfig, HotplugMethod, MemoryConfig, PayloadConfig,
-        RngConfig,
+    use crate::vm_config::DebugConsoleConfig;
+    use crate::vm_config::{
+        ConsoleConfig, ConsoleOutputMode, CpuFeatures, CpusConfig, HotplugMethod, MemoryConfig,
+        PayloadConfig, RngConfig,
     };
```

Comment:
Ah yes, then we could do this and format it. I believe this is what `imports_granularity` would do if we set it to `Crate` :)

```suggestion
    use crate::vm_config::{
        #[cfg(target_arch = "x86_64")] DebugConsoleConfig,
        ConsoleConfig, ConsoleOutputMode, CpuFeatures, CpusConfig, 
        HotplugMethod, MemoryConfig, PayloadConfig, RngConfig,
    };
```

Reply:
Cannot use conditional compilation within the `use` statement in rust 1.77 or 1.80.